test/unit/math/prim/fun/exp_test.cpp

```diff
@@ -3,8 +3,8 @@
 
 TEST(MathFunctions, expInt) {
   using stan::math::exp;
-  using std::exp;
```

Comment:
Can you double check the other function tests also have these removed?

Reply:
Sure.

Reply:
I did a grep of `using std::` in test/unit/math/prim/fun and apart from log, sqrt, exp the only uses are with std::vector and a few tests that explicitly want std::exp, std::log or std::fabs. I think we are good.